test/Microsoft.ML.Tests/CollectionDataSourceTests.cs

```diff
@@ -294,29 +294,13 @@ public class ConversionSimpleClass
             public float fFloat;
             public double fDouble;
             public bool fBool;
-            public string fString;
-        }
-
-        public class ConversionNullalbeClass
-        {
-            public int? fInt;
-            public uint? fuInt;
-            public short? fShort;
-            public ushort? fuShort;
-            public sbyte? fsByte;
-            public byte? fByte;
-            public long? fLong;
-            public ulong? fuLong;
-            public float? fFloat;
-            public double? fDouble;
-            public bool? fBool;
-            public string fString;
```

Comment:
I think we should still support `string` types, and as such, `string` can be `null`. So we should have some tests for null strings.

Reply:
What will null string map to? empty string?

---
In reply to: [215970412](https://github.com/dotnet/machinelearning/pull/705#discussion_r215970412) [](ancestors = 215970412)